### ISSUE TITLE
Fix minimum/exclusiveMinimum and maximum/exclusiveMaximum error messages

### ISSUE THREAD

**Issue by SebastianG77** [tdegrunt/jsonschema#274]
When validating against a schema that uses the property exclusiveMinimum, an inappropriate error message occurs. For the following example I get the error message: "instance must have a minimum value of 5". However, the validation error should be something like: "instance must have a minimum value greater than 5"

```
var Validator = require('jsonschema').Validator;
var v = new Validator();
var instance = 5;
var schema = {"type": "number", "minimum":5, "exclusiveMinimum": true};
console.log(v.validate(instance, schema).errors[0].stack); // instance must have a minimum value of 5
```

**Comment by awwright** [comment]
I'll work on this as a part of the next minor update adding number-form exclusiveMinimum/exclusiveMaximum support.

**Comment by awwright** [comment]
Fixed in b5a0b81236d4b42c27c99168c79faaa3a29ff7fd
